file-system.ts

```diff
@@ -55,6 +55,10 @@ export class FileSystem implements IFileSystem {
 		});
 	}
 
+	public utimes(path: string, atime: Date, mtime: Date): void {
+		return fs.utimesSync(path, atime, mtime);
```

Comment:
this is done as a guard against future behavior changes - in case the original `fs.utimesSync` returns something on previous or future versions of node.js we needn't change the code here